lib/stream-chat/client.rb

```diff
@@ -156,11 +184,16 @@ def review_flag_report(report_id, review_result, user_id, **details)
       patch("moderation/reports/#{report_id}", data: data)
     end
 
+    # Returns a message.
     sig { params(id: String).returns(StreamChat::StreamResponse) }
     def get_message(id)
       get("messages/#{id}")
     end
 
+    # Searches for messages.
+    #
+    # You can enable and/or disable the search indexing on a per channel
+    # type through the Stream Dashboard.
```

Comment:
there is something wrong with this sentence: "on a per channel"

Reply:
probably "per channel basis"

<img width="960" alt="image" src="https://user-images.githubusercontent.com/19969687/163392068-c24eb499-e0aa-47ce-8aa2-2e9bdd395f42.png">